docs/storage.md

```diff
@@ -137,19 +137,16 @@ To learn more about existing integrations with remote storage systems, see the [
 If a user wants to create blocks into the TSDB from data which is in [OpenMetrics](https://openmetrics.io/) format, they can do so using backfilling. However, they should be careful and note that it is not safe to backfill data from the last 3 hours (the current head block).
 
 
-The backfill sub-command is implemented such that it keeps at most 2 hours of data in the memory. Hence, it is important to make sure that there is enough RAM for 2 hours worth of data at a time.
+The backfill command is implemented such that Prometheus keeps at most 2 hours of data in the memory. 
```

Comment:
promtool, not Prometheus

Reply:
The idea it's trying to get across is that the memory usage of the tool is bounded, which I can see being a frequently asked question.

Reply:
Suggestion: "Backfilling will create new TSDB blocks, each containing two hours of metrics data. This limits the memory usage of block creation. Compacting the two hour blocks into larger blocks is later done by the Prometheus server itself."

Reply:
Suggestion looks good to me . Changed it